custom_components/unfoldedcircle/pyUnfoldedCircleRemote/remote.py

```diff
@@ -862,8 +862,7 @@ async def get_version(self) -> dict[str]:
             if self._is_simulator is True:
                 core = information.get("core", "")
                 # We only care about the beginning of the version for this compare
-                if Version(core[0:4]) >= Version("0.49"):
-                    self._external_entity_configuration_available = True
+                self._external_entity_configuration_available = True
```

Comment:
Was this causing an issue? It's not super important, but for older simulator releases, it's not supported.

Reply:
well on my remote simulator : I had to change this because I couldn't select entities. But my config is exotic : HA driver is configured as external. We could let this code don't you think ? now all the simulators should support configuration of entities